readthedocsext/theme/templates/includes/elements/promotions/ruby-support.html

```diff
@@ -0,0 +1,9 @@
+<div class="ui card">
+  <div class="content">
+    <a class="header" target="_blank" href="https://about.readthedocs.com/blog/2024/03/read-the-docs-loves-ruby/">Read the Docs ❤️ Ruby</a>
```

Comment:
In the blog, click bait titles are okayish, but in our application I feel we should be clear to the user:

```suggestion
    <a class="header" target="_blank" href="https://about.readthedocs.com/blog/2024/03/read-the-docs-loves-ruby/">{% trans "New: support for Ruby" %}</a>
```